examples/vl3_basic/Makefile

```diff
@@ -3,7 +3,7 @@ CONTAINERS = vl3-nse
 PODS = simple-client vl3-nse
 CHECK = export ITERATIONS=10 && export BATCHES=1 && scripts/check_vl3.sh
 
-#export VPP_AGENT=ligato/vpp-agent:v2.1.1
+VPP_AGENT = ligato/vpp-agent:v3.1.0
```

Comment:
just name this a different var and change line 29 to use it.

Reply:
ok, but I also defined `ORG` and `TAG` here because of the same 'bug', kiknos Makefile takes precedence and sets the `TAG` to kiknos